src/teleop/gui/src/components/JointStateTable.vue

```diff
@@ -0,0 +1,180 @@
+<template>
```

Comment:
I think a merge with master is in order, these tables should not have been changed at all by this PR...